css-four-column-layout/styles.css

```diff
@@ -38,7 +38,8 @@ p {
 }
 
 .col-1-4 {
-  margin: 0 -2.25rem;
+  flex-basis: 25%;
+  margin: 0 -1.5rem;
```

Comment:
This margin still needs to be on something separate. Column classes should only include `flex-basis` or `width`